sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -467,10 +494,148 @@ The `node` class provides the {crs}[common reference semantics].
 [source,c++]
 ----
 namespace sycl::ext::oneapi::experimental {
-  class node {};
+  class node {
+    template <int Dimensions>
+    void update_nd_range(nd_range<Dimensions> executionRange);
+  };
 }
 ----
 
+Table {counter: tableNumber}. Member functions of the `node` class.
+[cols="2a,a"]
+|===
+|Member Function|Description
+
+|
+[source,c++]
+----
+template <int Dimensions>
+void update_nd_range(nd_range<Dimensions> executionRange);
+----
+| Updates the ND-Range for this node with a new value. This new value will not
+affect any executable graphs this node is part of until it is passed to the
+executable graph's update function.
+See <<executable-graph-update, Executable Graph Update>> for more information
+about updating node parameters.
+
+Parameters:
+
+* `executionRange` - The new value for the ND-Range.
+
+Exceptions:
```

Comment:
We should possibly error here if we try to change dimension, see Ben Ashbaugh's comments in https://github.com/KhronosGroup/OpenCL-Docs/issues/1057#issuecomment-1944814490